web/src/pages/Cases/CaseDetails/Voting/VotesDetails/AccordionTitle.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { Answer } from "context/NewDisputeContext";
+import { responsiveSize } from "styles/responsiveSize";
+import { isUndefined } from "utils/index";
+import Identicon from "react-identicons";
+import { getVoteChoice } from "utils/getVoteChoice";
+import { shortenAddress } from "utils/shortenAddress";
+import { landscapeStyle } from "styles/landscapeStyle";
+
+const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  gap: ${responsiveSize(8, 12)};
+  flex-wrap: wrap;
+  ${landscapeStyle(
+    () => css`
+      flex-direction: row;
+      align-items: center;
+    `
+  )}
+`;
+const AddressContainer = styled.div`
+  display: flex;
+  gap: 4px;
+  align-items: center;
+`;
+const StyledLabel = styled.label<{ variant?: string }>`
+  color: ${({ theme, variant }) => (variant ? theme[variant] : theme.primaryText)};
+`;
+
+const VoteStatus: React.FC<{
+  choice?: string;
+  period: string;
+  answers: Answer[];
+  isActiveRound: boolean;
+}> = ({ choice, period, answers, isActiveRound }) => {
+  if (isUndefined(choice) && (isActiveRound ? ["appeal", "execution"].includes(period) : true))
+    return <StyledLabel>Forgot to vote</StyledLabel>;
```

Comment:
Maybe change this to:
```suggestion
    return <StyledLabel>Didn't cast a vote</StyledLabel>;
```
I think this is more explicative.